### PR TITLE
Add dateformat utility class

### PR DESCRIPTION
**Pull request by brandonpas**

### CODE REVIEW

**Comment by wellingtonmb88** [comment]
Good job, What do you think to create an UnitTest for this class?

**Comment by brandonpas** [comment]
Added unit tests for the DateTimeFormatUtil class.

File: app/src/androidTest/java/org/codedoesgood/androidtemplate/UtilityTests/Tests/DateTimeFormatTest.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Calendar;
+
+import static junit.framework.Assert.assertTrue;
+
+@RunWith(AndroidJUnit4.class)
+public class DateTimeFormatTest {
```
Comment (wellingtonmb88) [inline app/src/androidTest/java/org/codedoesgood/androidtemplate/UtilityTests/Tests/DateTimeFormatTest.java]: Good Test class but I believe it's better to use Roboeletric for Unit Test accessing the Android Framework, with Roboeletric the Test becomes faster. Site: http://robolectric.org/

File: app/src/androidTest/java/org/codedoesgood/androidtemplate/UtilityTests/Tests/DateTimeFormatTest.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    @Rule
+    public ActivityTestRule<MainActivity> mActivityRule =
```
Comment (wellingtonmb88) [inline app/src/androidTest/java/org/codedoesgood/androidtemplate/UtilityTests/Tests/DateTimeFormatTest.java]: Don't use Ungarian notation, remove the 'm'

**Pull request merged**